app/components/bali/avatar/component.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Bali
+  module Avatar
+    class Component < ApplicationViewComponent
+      renders_one :picture, Picture::Component
+
+      def initialize(form:,
+                     attr_model_name:,
+                     accepted_formats:,
```

Comment:
I think this should be an array and have a default value, since most of the time you will be uploading either jpg or png.

```suggestion
                     accepted_formats: [:jpg, :jpeg, :png],
```